resource/js/docready.js

```diff
@@ -524,8 +524,6 @@ $(function() { // DOCUMENT READY
     style: { classes: 'qtip-tipsy qtip-skosmos' }
   };
 
-  $('#navi4').qtip(qtip_skosmos);
-
```

Comment:
`#navi4` is the Help menu item. I did only this element before asking others for feedback here.

Reply:
Right, but there are also other UI elements where qtip is currently used, for example the lines just below this one.

Reply:
Yup. I did just one UI element first to see what others thought, as replacing all the other elements would take a bit longer and wouldn't be worth if the idea was discarded. I will check the rest of your feedback and once I've addressed all of it, I will start replacing in the other UI elements so that we can have a final preview :+1: